nix/modules/kld-toml-mapping.nix

```diff
@@ -27,7 +27,16 @@ in
     kuutamo.disko.disks = cfg.disks;
     kuutamo.disko.bitcoindDisks = cfg.bitcoind_disks;
 
-    users.extraUsers.root.openssh.authorizedKeys.keys = cfg.public_ssh_keys;
+    users.extraUsers = {
+      root.openssh.authorizedKeys.keys = cfg.public_ssh_keys;
+    } // lib.attrsets.mapAttrs
+      (n: v: {
+        isNormalUser = true;
+        group = n;
+        extraGroups = [ "kld" "cockroachdb" ];
```

Comment:
This gives too much power I think

Reply:
We can discuss this in the meeting, or we can create another group for these user.